tap_exacttarget/state.py

```diff
@@ -28,7 +28,7 @@ def get_last_record_value_for_table(state, table):
         return None
 
     date_obj = datetime.datetime.strptime(raw, DATE_FORMAT)
-    date_obj = date_obj - datetime.timedelta(days=1)
+    date_obj = date_obj - datetime.timedelta(hours=6)
```

Comment:
I am reluctant to make this "default" change. If we need to do this, I think the best course of action is to make this a sort of config change where users can pass in their desired time and the tap adjusts accordingly.